server/src/com/google/refine/Refine.java

```diff
@@ -487,7 +487,7 @@ public void actionPerformed(ActionEvent e) {
     }
 
     private void openBrowser() {
-        if (!Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
+        if (!Desktop.getDesktop().isSupported(Desktop.Action.BROWSE) || !(System.getenv("SNAP") == null)) {
```

Comment:
Quick simplification:
```suggestion
        if (!Desktop.getDesktop().isSupported(Desktop.Action.BROWSE) || System.getenv("SNAP") != null) {
```